phpunit.xml.dist

```diff
@@ -0,0 +1,18 @@
+<phpunit
```

Comment:
co pridat i
```
convertErrorsToExceptions   = "true"
convertNoticesToExceptions  = "true"
convertWarningsToExceptions = "true"
```

Reply:
To je default...

https://phpunit.de/manual/current/en/appendixes.configuration.html
http://schema.phpunit.de/6.4/phpunit.xsd